test/go.test.ts

```diff
@@ -406,52 +406,52 @@ It returns the number of bytes written and any write error encountered.
 		}).then(() => done(), done);
 	});
 
-	// test('Gometalinter error checking', (done) => {
-	// 	getGoVersion().then(version => {
-	// 		if (version && version.major === 1 && version.minor < 6) {
-	// 			// golint in gometalinter is not supported in Go 1.5, so skip the test
-	// 			return Promise.resolve();
-	// 		}
-
-	// 		let config = Object.create(vscode.workspace.getConfiguration('go'), {
-	// 			'lintOnSave': { value: 'package' },
-	// 			'lintTool': { value: 'gometalinter' },
-	// 			'lintFlags': { value: ['--disable-all', '--enable=varcheck', '--enable=errcheck'] },
-	// 			'vetOnSave': { value: 'off' },
-	// 			'buildOnSave': { value: 'off' }
-	// 		});
-	// 		let expected = [
-	// 			{ line: 11, severity: 'warning', msg: 'error return value not checked (undeclared name: prin) (errcheck)' },
-	// 			{ line: 11, severity: 'warning', msg: 'unused variable or constant undeclared name: prin (varcheck)' },
-	// 		];
-	// 		let errorsTestPath = path.join(fixturePath, 'errorsTest', 'errors.go');
-	// 		return check(vscode.Uri.file(errorsTestPath), config).then(diagnostics => {
-	// 			const allDiagnostics = [].concat.apply([], diagnostics.map(x => x.errors));
-	// 			let sortedDiagnostics = allDiagnostics.sort((a, b) => {
-	// 				if (a.msg < b.msg)
-	// 					return -1;
-	// 				if (a.msg > b.msg)
-	// 					return 1;
-	// 				return 0;
-	// 			});
-
-	// 			assert.equal(sortedDiagnostics.length > 0, true, `Failed to get linter results`);
-	// 			let matchCount = 0;
-	// 			for (let i in expected) {
-	// 				for (let j in sortedDiagnostics) {
-	// 					if ((expected[i].line === sortedDiagnostics[j].line)
-	// 						&& (expected[i].severity === sortedDiagnostics[j].severity)
-	// 						&& (expected[i].msg === sortedDiagnostics[j].msg)) {
-	// 						matchCount++;
-	// 					}
-	// 				}
-	// 			}
-	// 			assert.equal(matchCount >= expected.length, true, `Failed to match expected errors`);
-
-	// 			return Promise.resolve();
-	// 		});
-	// 	}).then(() => done(), done);
-	// });
+	test('Gometalinter error checking', (done) => {
+		getGoVersion().then(version => {
```

Comment:
i guess needs a rebase on master since this is checked in ?

Reply:
oh actually the fix was checked but this was commented out, all good then

---
In reply to: [253680465](https://github.com/Microsoft/vscode-go/pull/2294#discussion_r253680465) [](ancestors = 253680465)